### DISCUSSION TITLE
integrate paystack gateway into sketchware project

### DISCUSSION THREAD

**Discussion by Candid-emeka** [Sketchware-Pro/Sketchware-Pro#831] [Q&A]
i want to know if i can integrate paystack gateway to my sketchware app project, so i can easily receive payments from the app using sketchware, i have tried some several options but still encounterrssome error. 
please if any one can help me out i will be grateful

**Comment by PranavPurwar** [comment]
Use discord server for help with projects. No more components/views will be added to sketchware pro because we can't add each and everything in app. Please check the respective library's documentation.
